installer/scripts/omsadmin.sh

```diff
@@ -608,6 +649,13 @@ remove_all()
     # Remove LAD workspace
     WORKSPACE_ID="LAD"
     remove_workspace
+
+    # remove scom workspace
+    ls -l $ETC_DIR | grep -w scom > /dev/null 2> /dev/null
```

Comment:
I am using ls -1 (digit 1) here that only lists prints the name of files present. Then I grep for word scom to find if som workspace is present or not.